Documentation/using-the-client.md

```diff
@@ -75,11 +75,16 @@ Start and stop units with the `start` and `stop` commands:
 $ fleetctl start goodbye.service
 Unit goodbye.service launched on 85c0c595.../172.17.8.102
 
+$ fleetctl restart goodbye.service
```

Comment:
Please move the restart-related documentation to the bottom of the "Start and stop units" block.